lib/proxy/ip-forward.php

```diff
@@ -144,18 +144,31 @@ function set_remote_address( $ip ) {
 	$_SERVER['REMOTE_ADDR'] = $ip;
 }
 
+if ( ! function_exists( __NAMESPACE__ . '\\_get_wpcom_vip_proxy_verification' ) ) {
+	/**
+	 * @access private
+	 * @internal
+	 * @return null|string 
+	 */
+	function _get_wpcom_vip_proxy_verification(): ?string {
+		if ( defined( 'WPCOM_VIP_PROXY_VERIFICATION' ) && ! empty( WPCOM_VIP_PROXY_VERIFICATION ) ) {
+			return (string) WPCOM_VIP_PROXY_VERIFICATION;
+		}
+
+		return null;
+	}
+}
```

Comment:
This change allows us to override `_get_wpcom_vip_proxy_verification()` in our test suite; the overridden function does not have to use a constant, it can be much more flexible.